metrix-mapping/src/main/groovy/com/powsybl/metrix/mapping/TimeSeriesDslLoader.groovy

```diff
@@ -76,13 +77,13 @@ class TimeSeriesDslLoader {
         logDslLoader.logWarn(message)
     }
 
-    protected static List<String> getStaticStars() {
+    protected List<String> getStaticStars() {
         List<String> staticStars = new ArrayList<>()
-        staticStars.add("com.powsybl.metrix.mapping.SimpleEquipmentGroupType")
+        staticStars.add(equipmentGroupTypes)
         return staticStars
     }
 
-    private static CompilerConfiguration createCompilerConfig() {
```

Comment:
createCompilerConfig can remain static

Reply:
It won't work since createCompilerConfig calls getStaticStars (which would not be static)

Reply:
of course